src/test/java/uk/gov/hmcts/reform/unspec/sampledata/CaseDataBuilder.java

```diff
@@ -645,55 +645,116 @@ public CaseDataBuilder atStatePaymentFailed() {
     }
 
     public CaseDataBuilder atStatePaymentSuccessful() {
-        atStatePendingCaseIssued();
+        atStateClaimSubmitted();
         claimIssuedPaymentDetails = PaymentDetails.builder()
             .status(SUCCESS)
             .reference("RC-1604-0739-2145-4711")
             .build();
+        paymentReference = "12345";
         paymentSuccessfulDate = LocalDateTime.now();
-        claimDetailsNotificationDeadline = LocalDateTime.now().plusDays(1);
         return this;
     }
 
-    public CaseDataBuilder atStateAwaitingCaseNotification() {
+    public CaseDataBuilder atStatePendingClaimIssued() {
         atStatePaymentSuccessful();
-        ccdState = CASE_ISSUED;
         issueDate = CLAIM_ISSUED_DATE;
+        return this;
+    }
+
+    public CaseDataBuilder atStatePendingClaimIssuedUnRegisteredDefendant() {
+        atStatePaymentSuccessful();
+        issueDate = CLAIM_ISSUED_DATE;
+        respondent1Represented = YES;
+        respondent1OrgRegistered = NO;
+        return this;
+    }
+
+    public CaseDataBuilder atStatePendingClaimIssuedUnRepresentedDefendant() {
+        atStatePaymentSuccessful();
+        issueDate = CLAIM_ISSUED_DATE;
+        respondent1Represented = NO;
+        return this;
+    }
+
+    public CaseDataBuilder atStateClaimIssued() {
+        atStatePendingClaimIssued();
         claimNotificationDeadline = LocalDateTime.now();
+        ccdState = CASE_ISSUED;
         return this;
     }
 
-    public CaseDataBuilder atStateAwaitingCaseDetailsNotification() {
-        atStateAwaitingCaseNotification();
-        claimNotificationDate = LocalDateTime.now();
+    public CaseDataBuilder atStateClaimNotified() {
+        atStateClaimIssued();
+        claimNotificationDate = LocalDate.now().atStartOfDay();
         claimDetailsNotificationDeadline = DEADLINE;
         ccdState = AWAITING_CASE_DETAILS_NOTIFICATION;
         return this;
     }
 
-    public CaseDataBuilder atStateClaimCreated() {
-        atStateAwaitingCaseDetailsNotification();
+    public CaseDataBuilder atStateClaimDetailsNotified() {
+        atStateClaimNotified();
         claimDetailsNotificationDate = LocalDateTime.now();
         claimDismissedDeadline = LocalDateTime.now().plusMonths(6);
-        ccdState = AWAITING_RESPONDENT_ACKNOWLEDGEMENT;
         respondent1ResponseDeadline = RESPONSE_DEADLINE;
+        ccdState = AWAITING_RESPONDENT_ACKNOWLEDGEMENT;
         return this;
     }
 
-    public CaseDataBuilder atStateExtensionRequested() {
-        atStateClaimAcknowledge();
-        respondentSolicitor1AgreedDeadlineExtension = LocalDate.now();
+    public CaseDataBuilder atStateClaimDetailsNotifiedTimeExtension() {
+        atStateClaimDetailsNotified();
+        respondent1ResponseDeadline = RESPONSE_DEADLINE;
         respondent1TimeExtensionDate = LocalDateTime.now();
```

Comment:
I think we should set `respondentSolicitor1AgreedDeadlineExtension = LocalDate.now();` in this builder as well. what do you think?

Reply:
RESPONSE_DEADLINE is set as now() + 14 days, which sound bit better as issue date is set as now()

Reply:
I meant we need to set some value for respondentSolicitor1AgreedDeadlineExtension whether it’s now() or something else